app/src/prisma-client.ts

```diff
@@ -3,4 +3,6 @@ import { PrismaClient } from '@prisma/client'
 // This is experimental. It is used in /regionen/slug/stats (page RSC).
 // The idea is to use it with prismaClientForRawQueries.$queryRaw``
 // But only in situations where permissions are already handled by Blitz.
```

Comment:
```suggestion
// This is experimental. It is used in /regionen/slug/stats (page RSC).
// The idea is to use it with prismaClientForRawQueries.$queryRaw``
// But only in situations where permissions are already handled by Blitz.
```

@rush42 bitte den code comment updaten; er passt nicht mehr, weil es ja jetzt mehr als ein experiment ist und das schema thema hinzu kommt.